includes/functions/functions_lookups.php

```diff
@@ -290,3 +290,43 @@ function zen_get_order_status_name(int $order_status_id, int $language_id = 0)
     return $result->fields['orders_status_name'] . ' [' . (int)$order_status_id . ']';
 }
 
+
+function zen_lookup_admin_menu_language_override(string $lookup_type, string $lookup_key, string $fallback): string
+{
+    switch ($lookup_type) {
+        case 'order_status_name':
+            $lookup = strtoupper('ORDER_STATUS_NAME_FOR_' . $lookup_key);
+            break;
+        case 'product_type':
```

Comment:
@piloujp After looking at this more, I think we should rename this one lookup_type from `product_type` to `product_type_name`. It feels more logical/intuitive. Unfortunately it means updating the places where it's been called.

Reply:
@drbyte I have just sent a simplified version without all plugin stuffs and few modifications per your remarks.
I agree something should be done for internationalization on plugin side rather than in ZC files.